Task2/Task2/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите длину ребра куба");
+            double a = double.Parse(Console.ReadLine());
+
+            double v = (a*a*a);
+            double d = (a * Math.Sqrt(3));
```

Comment:
Лучше не ставить лишние скобки. Они затрудняют чтение кода. А вот отделение знаков операций пробелами облегчает. 
```suggestion
            double v = a * a * a;
            double d = a * Math.Sqrt(3);
```